corgidrp/data.py

```diff
@@ -216,6 +220,17 @@ def __init__(self, data_or_filepath, pri_hdr=None, ext_hdr=None, err = None, dq
                 else:
                     self.dq = np.zeros(self.data.shape, dtype = int)
 
+                if bias is not None:
+                    if (np.shape(self.data)[0],) != np.shape(bias):
+                        raise ValueError("The shape of bias is {0} while we are expecting shape {1}".format(bias.shape, self.data.shape))
+                    self.bias = bias
+                # we assume that the bias extension is index 4 of hdulist
+                elif len(hdulist)>4:
+                    self.bias = hdulist[4].data
+                    self.bias_hdr = hdulist[4].header
+                else:
+                    self.bias = np.zeros(self.data.shape[0], dtype = np.float32)
```

Comment:
I forget the specifications now. Are we purposely saving it as float32 for a reason here?

Reply:
I remember someone suggesting explicitly that we save the bias as an array of single-precision floats, maybe to reduce the data size, but I can't find exactly where the recommendation came from. If it doesn't matter, I can remove it!

Reply:
Ok, I see it in the specifications now. We actually haven't downsized the float/int size for the other HDUs, but I guess we can start here. Let's keep it then.